.gitignore

```diff
@@ -0,0 +1 @@
+/venv/
```

Comment:
Don't you use IDE? 
Pycharm adds `.idea/` directory. You can also add additional files or folder here. I usually just use this template
https://github.com/github/gitignore/blob/main/Python.gitignore

Reply:
Also, you need to add `files/` into .gitignore